src/components/webview/FileExplorerProvider.ts

```diff
@@ -100,6 +94,48 @@ export class FileExplorerProvider implements WebviewViewProvider {
 		});
 	}
 
+	public updateExplorerView(caseHash: string | null) {
```

Comment:
`caseHash: CaseHash`

`null` should be not accepted

Reply:
addressed